test/src/test/java/hudson/model/FreestyleJobPublisherTest.java

```diff
@@ -0,0 +1,89 @@
+package hudson.model;
+
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.model.utils.AbortExceptionPublisher;
+import hudson.model.utils.IOExceptionPublisher;
+import hudson.model.utils.ResultWriterPublisher;
+import hudson.model.utils.TrueFalsePublisher;
+import hudson.tasks.ArtifactArchiver;
+import org.apache.commons.io.FileUtils;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Freestyle publishers statuses tests
+ * @author Kanstantsin Shautsou
+ */
+public class FreestyleJobPublisherTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    /**
+     * Execute even one of publishers return false. Shows JENKINS-26964 bug.
+     */
+    @Issue("JENKINS-26964")
```

Comment:
You might want to move `@Issue` up to be a class annotation; or else copy it on the other test cases, assuming they are also manifestations of the bug (or controls).